### PR TITLE
Revert "Fix color contrast"

### PR DESCRIPTION
**Pull request by estelle**
Reverts mdn/content#23569

box shadows don't need to match WCAG text color contrasts. About 15 people have attempted to alter this example over the past 6 months because the colors are the colors of the trans flag.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
In this case, maybe either add an editorial note (Markdown comment) or an in-code comment that it's intended and there's no a11y issue?

File: files/en-us/web/css/box-shadow/index.md [hunk 1]
```diff
@@ -156,12 +156,13 @@ We added a margin the size of the widest box-shadow to ensure the shadow doesn't
...
 p {
-   box-shadow: 0 0 0 2em #2E3436,
-   0 0 0 4em #EDD400;
+  box-shadow: 0 0 0 2em #f4aab9, 
+     0 0 0 4em #66ccff;
   margin: 4em;
   padding: 1em;
 }
 ```
+<!-- these colors are intentionally pink and blue. WCAG requires color contrast between text and background, not between border colors. -->
```
Comment (Josh-Cena) [inline files/en-us/web/css/box-shadow/index.md:165]: Prettier says:

~~~md
```css
p {
  box-shadow: 0 0 0 2em #f4aab9, 0 0 0 4em #66ccff;
  margin: 4em;
  padding: 1em;
}
```

<!-- these colors are intentionally pink and blue. WCAG requires color contrast between text and background, not between border colors. -->
~~~

Comment (estelle) [reply files/en-us/web/css/box-shadow/index.md]: In the future, can we set prettier to allow page breaks on commas in CSS? It makes it easier to teach if each shadow, background-image (especially gradients), animation, etc., are on a new line.

**Review by Josh-Cena — APPROVED** [review]
🏳️‍⚧️

**Pull request merged**